Gemfile

```diff
@@ -130,6 +130,8 @@ group :development, :test do
   gem 'bullet', '>= 4.14.9'
 
   gem 'parallel_tests'
+  gem 'launchy'
+  gem 'pry-byebug'
```

Comment:
I'm okay with any tool that someone is familiar with, cause now I use Rubymine or byebug mostly, but it's still manageable for me 😛